src/helpers/url.js

```diff
@@ -81,10 +81,19 @@ const getDocumentUrlForFile = (fileDir, fileId) => {
 		})
 }
 
+export const getConfigFileUrl = () => {
+	return getCallbackBaseUrl() + '/index.php/apps/richdocuments/wopi/settings'
```

Comment:
Please use [generateUrl()](https://nextcloud-libraries.github.io/nextcloud-router/functions/generateUrl.html) as otherwise the path would not work for Nextcloud instances running in subdirectories.